xds/src/main/java/io/grpc/xds/client/XdsClientImpl.java

```diff
@@ -957,6 +960,7 @@ public void handleStreamClosed(Status status, boolean shouldTryFallback) {
 
       ControlPlaneClient cpcClosed = serverCpClientMap.get(serverInfo);
       if (cpcClosed == null) {
+        logger.log(XdsLogLevel.DEBUG, "Couldn't find CPC for {0}", serverInfo);
```

Comment:
This log statement is not at all clear. Any time this happens we'd expect us not to care because the CPC is shut down, but if you just read the message you would probably think this a surprise.

Reply:
I've made the message clearer.  We might care because cleanup and metric reporting are being skipped.